web/themes/contrib/civictheme/includes/link.inc

```diff
@@ -23,12 +22,14 @@ function _civictheme_process_html_content_urls_to_links(string $html): string {
     return $html;
   }
 
-  // Convert URLs to links using core filter_url.
-  $filter_manager = \Drupal::service('plugin.manager.filter');
-  $filter_collection = new FilterPluginCollection($filter_manager, []);
-  $filter = $filter_collection->get('filter_url');
+  // Use Drupal's text format processing system.
+  $build = [
+    '#type' => 'processed_text',
+    '#text' => $html,
+    '#format' => 'civictheme_rich_text',
+  ];
 
-  return _filter_url($html, $filter);
+  return (string) \Drupal::service('renderer')->render($build);
```

Comment:
@richardgaunt using this filter removes all HTML tags not allowed by the filter. 
Test failing as. figure is stripped off 
<img width="773" height="89" alt="Screenshot 2025-07-11 at 10 04 56 AM" src="https://github.com/user-attachments/assets/8a882f93-cd0e-4de1-bb2b-f0c20c73a50c" />